raiden/transfer/channel.py

```diff
@@ -244,6 +270,15 @@ def is_transfer_expired(
     return has_lock_expired
 
 
+def is_withdraw_expired(block_number: BlockNumber, expiration_threshold: BlockExpiration) -> bool:
+    """ Determine whether a withdraw has expired.
+
+    The withdraw has expired if the current block exceeds
+    the withdraw's expiration + confirmation blocks.
+    """
+    return block_number >= expiration_threshold
```

Comment:
The doc string doesn't match the code. 'Exceeds' suggests `>` instead of `>=`.

Reply:
Maybe "exceeds" should be replaced with "reaches and goes beyond the withdraw expiration".

But the operator is correct, once the expiration block is reached, the withdraw should be considered expired.